osgi.specs/docbook/core/058/service.resolver.xml

```diff
@@ -1415,6 +1416,144 @@ public List&lt;Capability&gt; findProviders( Requirement requirement) {
     </section>
   </section>
 
+  <section>
+    <title>Obtaining Resolvers Using SPI</title>
+
+    <para>While the <xref linkend="org.osgi.service.resolver.Resolver"
+    xrefstyle="hyperlink"/> service is typically obtained from the OSGi
+    service registry when running within an OSGi framework, there are many
+    scenarios where a resolver is needed outside an OSGi runtime:</para>
+
+    <itemizedlist>
+      <listitem>
+        <para><emphasis>Build Tools</emphasis> - Build tools such as Maven,
+        Gradle, or bnd need to resolve project dependencies at build time,
+        often modeling non-OSGi artifacts (like Maven dependencies) as
+        resources with requirements and capabilities.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis>IDEs and Development Tools</emphasis> - Integrated
+        development environments need to analyze bundle dependencies and
+        predict resolution outcomes without running a full OSGi
+        framework.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis>Provisioning Tools</emphasis> - Tools that operate on
+        repositories to download and provision bundles often need resolution
+        capabilities but do not themselves run within OSGi.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis>Feature Launchers</emphasis> - Tools that construct
+        and launch OSGi frameworks based on feature definitions (see <xref
+        linkend="service.featurelauncher"/>) need to resolve features before
+        the framework starts.</para>
+      </listitem>
```

Comment:
I would avoid a reference to the feature launcher from the core specification.  I don't think it is necessary and the Feature Launcher is a specified provisioning tool so should be covered by the Provisioning Tool. I want to avoid links to other specifications that are not even published yet from the Core spec.